src/index.css

```diff
@@ -11,3 +11,113 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+body {
+  background-color: black;
+}
+
+.header {
+  color: #fff;
+  display: flex;
+  font-family: Helvetica, sans-serif;
+  font-size: 20px;
+  justify-content: center;
+  margin-bottom: 2em;
+  /*border: 2px solid red;*/
+}
+
+/*-----wrapping all albums-----*/
+.album-wrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 25px;
+  /* border: 2px solid red;*/
+}
+
+/*----------wrapping every single album-----*/
+.album {
+  margin: 10px;
+  color: #fff;
+  font-size: 14px;
+  /*border: 2px solid red;*/
+}
+
+.artistName h3 {
+  color: #a0a0a0;
+}
+
+/*--------- container to albim image and buttons-----*/
+.cover-image-container {
+  position: relative;
+  width: 100%;
+  /*border: 4px solid green;*/
+}
+
+img {
+  width: 100%;
+  height: auto;
+  max-width: 600px;
+  transition: opacity 0.3s ease;
+}
+
+.buttons {
+  display: none;
+  position: absolute;
+  top: 50%;
+  left: 40%;
+
+  width: 135px;
+  fill: #fff;
+}
+
+/* Style the buttons */
+.buttons button {
+  background: none;
+  border: none;
+  padding: 0;
+  margin: 0;
+  cursor: pointer;
+  /* fill: #fff; */
+}
+
+/* Style the icons inside the buttons */
+.buttons button svg {
```

Comment:
Don't know if it would work, but maybe try:
.buttons > button > svg {

or  just

svg:hover {